bcl/src/bc_radio.c

```diff
@@ -711,3 +711,49 @@ static bool _bc_radio_peer_device_remove(uint64_t device_address)
 
 	return false;
 }
+
+
+void _bc_radio_button_event_handler(bc_button_t *self, bc_button_event_t event, void *event_param)
+{
+    (void) self;
+    (void) event_param;
+    bc_led_t *led = (bc_led_t*)event_param;
+
+    if (event == BC_BUTTON_EVENT_PRESS)
+    {
+        if(led)
+        {
+            bc_led_pulse(led, 100);
+        }
+
+        static uint16_t event_count = 0;
+
+        bc_radio_pub_push_button(&event_count);
+
+        event_count++;
+    }
+    else if (event == BC_BUTTON_EVENT_HOLD)
+    {
+        bc_radio_enroll_to_gateway();
+
+        if(led)
+        {
+            bc_led_set_mode(led, BC_LED_MODE_OFF);
+            bc_led_pulse(led, 1000);
+        }
+    }
+}
+
+void bc_radio_button_pair(bc_button_t *button, bc_led_t *led)
+{
+    // Init button and callback
+    bc_button_init(button, BC_GPIO_BUTTON, BC_GPIO_PULL_DOWN, false);
```

Comment:
It is not `false` the last parameter but `0` ... we have switched from true/false to int mechanism for GPIO states